dateparser/data/date_translation_data/en.py

```diff
@@ -280,10 +280,10 @@
             "(\\d+) decades? ago"
         ],
         "in \\1 century": [
-            "in (\\d+) century?"
+            "in (\\d+) centurys?"
```

Comment:
I think it needs to be something like `centur(?:ys?|ies)`

Reply:
Just tried this , Tests are failing when I used it.